### PR TITLE
Move all Dockerfiles to arch-specific dir

### PR DESCRIPTION
**Pull request by NikolaMilosavljevic**
Fixes: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/87

Moving all Dockerfiles that are not in arch-specific directory to a new arch directory. Keeping Dockerfiles in directories that include architecture in the directory name.

### CODE REVIEW

**Comment by NikolaMilosavljevic** [comment]
Working on some missing updates in manifest files.

File: src/cbl-mariner/manifest.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
             },
             {
               "architecture": "arm64",
-              "dockerfile": "src/cbl-mariner/2.0/docker-testrunner",
+              "dockerfile": "src/cbl-mariner/2.0/docker-testrunner/amd64",
```
Comment (mthalman) [inline src/cbl-mariner/manifest.json:23]: As you can see from the `architecture` property, this is `arm64` but is also used for `amd64` builds. In other words, this is a multi-arch Dockerfile. So it shouldn't be placed in an architecture-specific directory.

File: src/raspbian/manifest.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
           "platforms": [
             {
               "architecture": "arm",
-              "dockerfile": "src/raspbian/10/crossdeps",
+              "dockerfile": "src/raspbian/10/crossdeps/arm32v6",
```
Comment (mthalman) [inline src/raspbian/manifest.json:10]: You can see in the `variant` property below that this should be `v7`.

File: src/cbl-mariner/manifest.json (hunk 1 above)
Comment (NikolaMilosavljevic) [reply src/cbl-mariner/manifest.json:23]: Will fix - thanks!

Comment (NikolaMilosavljevic) [reply src/cbl-mariner/manifest.json]: Fixed with https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1047/commits/8fce682c040a27b7d2ae70ef936d3b0f84434127

**Comment by mthalman** [comment]
Looks like there are some manifest conflicts to resolve.

**Comment by NikolaMilosavljevic** [comment]
> Looks like there are some manifest conflicts to resolve.

Will update file paths as well, if needed.

**Comment by NikolaMilosavljevic** [comment]
> > Looks like there are some manifest conflicts to resolve.
> 
> Will update file paths as well, if needed.

Fixed with https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1047/commits/4496739b046552b89c38ffc25c73a543ac29ad0c

**Pull request merged**